app/assets/v2/js/grants/fund.js

```diff
@@ -541,7 +526,7 @@ const subscribeToGrant = (transactionHash) => {
           web3.utils.toChecksumAddress(selected_token), // token denomination / address
           web3.utils.toTwosComplement(Number(grant_amount * Math.pow(10, decimals)).toLocaleString('fullwide', {useGrouping: false})),
           web3.utils.toTwosComplement(realPeriodSeconds),
-          web3.utils.toTwosComplement(data.gas_price),
+          web3.utils.toTwosComplement(1),
```

Comment:
TODO: check why subminer fails

Reply:
do you want a fast and expensive gas or a slow and cheap one? or in the middle?